src/routes/user/postRoutes.ts

```diff
@@ -1,40 +1,15 @@
-import { postController } from '@/controllers/postController';
-import { checkAdmin } from '@/middlewares/authValidation';
-import { validateUserId } from '@/middlewares/userValidation';
-import { validatePostBody } from '@/middlewares/postValidations';
-import { Hono } from 'hono';
-import { jwt } from 'hono/jwt';
-
-const secretKey = process.env.SECRET_KEY!;
+import { postController } from "@/controllers/postController";
+import { validateUserId } from "@/middlewares/userValidation";
+import {
+  validatePostBody,
+  validateUpdatePostBody,
+} from "@/middlewares/postValidations";
+import { Hono } from "hono";
 
-export const postRoutes = new Hono().basePath('/post');
+export const postRoutes = new Hono().basePath("/post");
 
```

Comment:
Why did you removed the jwt middleware?

Reply:
because of jwt in userRoutes, it is unused